src/lib/token_list_gen.ts

```diff
@@ -45,6 +45,7 @@ export const generateTokenList = async (
   l1TokenList: TokenList,
   prevArbTokenList?: ArbTokenList,
   options?: {
+    includeAllL1Tokens?: boolean,
```

Comment:
`includeAllL1Tokens` means that it will maintain the original list, right?
we could maybe name this `appendArbifiedToOriginalList` or something
I had to scratch my head a bit to understand, else some comments could also help

Reply:
Yeah, I'll add some comments. This was just to follow the naming convention of `includeUnbridgedL1Tokens`.